inspire_query_parser/visitors/elastic_search_visitor.py

```diff
@@ -166,7 +166,7 @@ class ElasticSearchVisitor(Visitor):
     AUTHORS_NAME_VARIATIONS_FIELD = 'authors.name_variations'
     AUTHORS_BAI_FIELD = 'authors.ids.value'
     BAI_REGEX = re.compile(r'^((\w|-|\')+\.)+\d+$', re.UNICODE | re.IGNORECASE)
-    TEXKEY_REGEX = re.compile(r'^\w+:\d{4}\w{2,3}', re.UNICODE)
+    TEXKEY_REGEX = re.compile(r'^\w+:\d{4}[a-z]{2,3}', re.UNICODE)
```

Comment:
can you also make the first part more strict?
```suggestion
    TEXKEY_REGEX = re.compile(r'[a-zA-Z-]+:\d{4}[a-z]{2,3}', re.UNICODE)
```